### ISSUE TITLE
Ben needs a haircut

### ISSUE THREAD

**Issue by JoshTheDeveloperr** [benawad/astral-bugs#17]
Here you go m8 https://booksy.com/en-us

**Comment by resqiar** [comment]
Definitely

**Comment by AbdallahMostafaIbrahim** [comment]
I agree

**Comment by FnrDev** [comment]
I agree

**Comment by CAPSLOCKFURY** [comment]
Nope, with his current haircut he looks exactly like no-life billion dollars startup developer

**Comment by resqiar** [comment]
Ben should consider Zuck's haircut tho

**Comment by YassinEldeeb** [comment]
Yeah that issue needs to be resolved 😂

**Comment by ghost** [comment]
his hairs fine
also btw he gets notified whenever issues are opened on his repos same as anybody else and i rly dont think he appreciates people using it as a way to try to talk to him or tell him things not related to the repo itself

**Comment by ghost** [comment]
maybe its helpful idk
he hasnt closed it yet so he couldve actually benefited from it
